hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -203,11 +221,10 @@ protected List<HoodieFileGroup> buildFileGroups(Stream<HoodieBaseFile> baseFileS
       if (baseFiles.containsKey(pair)) {
```

Comment:
`baseFiles` need to be sorted in rever order or instant (start) time as per the algorithm mentioned in RFC. Where are we ensuring that?

Reply:
The fileslices map is a tree map, which already sorts the base instant time in reverse order.